services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/Application.kt

```diff
@@ -13,5 +13,6 @@ data class Application(
     val alluStatus: ApplicationStatus?,
     val applicationIdentifier: String?,
     val applicationType: ApplicationType,
-    val applicationData: ApplicationData
+    val applicationData: ApplicationData,
+    val hankeTunnus: String?,
```

Comment:
Why is the hankeTunnus nullable? Shouldn't an application be always linked to a hanke?